doc/client.rst

```diff
@@ -1071,7 +1071,7 @@ user\'s roles. Users are assigned roles, which are collections of \
         print("Error [{0}]: {1}".format(e.code, e.msg))
     client.close()
 
-.. seealso:: `Security features article <https://docs.aerospike.com/server/guide/security/index.html>`_.
+.. seealso:: `Security features article <https://aerospike.com/docs/server/guide/security/index.html>`_.
```

Comment:
This link returns a 404

Reply:
Should be https://aerospike.com/docs/server/guide/security